packages/core/src/RPA/core/logger.py

```diff
@@ -11,6 +12,15 @@
 from RPA.core.helpers import required_param
 
 
+def deprecation(message):
+    """Emits once a deprecation warning log given the provided `message`."""
+    logging.captureWarnings(True)
+    try:
+        warnings.warn(message, FutureWarning, stacklevel=2)
+    finally:
+        logging.captureWarnings(True)
```

Comment:
Should this be `False`?

Reply:
Yes, dumb mistake :) I made.
Updated, also in the latest `6.5.1` hotfix of **rpaframework-core**.